docs/Releasing-Mock.md

```diff
@@ -85,6 +57,8 @@ helps you to setup a correct configuration layout.
        $ git log mock-4.1-1..HEAD --format="%aN" mock/ | sort | uniq
        $ git log mock-core-configs-38.1-1..HEAD --format="%aN" mock-core-configs/ | sort | uniq
 
+6. Commit all the pending changes
```

Comment:
Nit, this should be done before we check the latest commit CI status.

Reply:
I meant commit all changes generated by towncrier